File: simple/csharp/extract-variable_before2.cs
void renderBanner() 
{
  // When all the parameters are set, counted 
  // the boundaries of the frame, we print the 
  // result on the screen.
  if (((frame.isChanged() || target.isChanged()) || 
        experiment.isRunning()) &&
        frame.getSize().equal(screen.getSize())) 
  {
    // print
  }
}

File: simple/php/extract-variable_before2.php
void renderBanner() {
  // When all the parameters are set, counted 
  // the boundaries of the frame, we print the 
  // result on the screen.
  if ((($frame->isChanged() or $target->isChanged()) or 
        $experiment->isRunning()) and
        $frame->getSize()->equal($screen->getSize())) {
    // print
  }
}
